# 制御ループ周波数
hz: 10              # ループ周波数[Hz]

# 目標軌道
radius: 3.0         # 軌道の旋回半径[m]
cource_length: 30   # x軸方向の長さ[m]
resolution: 0.1     # 軌道生成時の刻み幅[m]

# 初期位置
start_point_x: 0.0  # x座標[m]
start_point_y: 0.0  # y座標[m]
